spec/unit/shoulda/matchers/active_record/have_rich_text_matcher_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveRichText, type: :model do
+  def self.rich_text_is_defined?
+    defined?(ActionText::RichText)
+  end
+
+  if rich_text_is_defined?
+    context 'when the model has a RichText association' do
+      it 'matches when the subject configures has_rich_text' do
+        valid_record = new_post(is_rich_text_association: true)
+
+        expected_message = 'Did not expect Post to have ActionText::RichText :content'
+
+        expect { have_rich_text(:content) }.
+          to match_against(valid_record).
+          or_fail_with(expected_message)
+      end
+
+      it 'returns the message including the name of the provided association' do
+        matcher = have_rich_text(:content)
+        expect(matcher.description).
+          to eq('have configured :content as a ActionText::RichText association')
+      end
+    end
+
+    context 'when the model does not have a RichText association' do
+      it 'does not match when provided with a model attribute' do
```

Comment:
In both of these tests, we are providing a model attribute to the matcher. It seems like the thing that's different between these two tests is that in one, the attribute being tested exists on the model, whereas in the other, it does not. What if you were to have the names of these tests reflect that?